api/src/main/java/org/pmiops/workbench/exfiltration/EgressLogService.java

```diff
@@ -67,8 +70,10 @@ public EgressLogService(
   public List<AuditEgressRuntimeLogGroup> getRuntimeLogGroups(DbEgressEvent event) {
     Duration windowSize = Duration.ofSeconds(event.getEgressWindowSeconds());
     Instant endTime = event.getCreationTime().toInstant();
+    Optional<SumologicEgressEvent> maybeSumologicEvent =
+        maybeParseSumologicEvent(event.getSumologicEvent());
     Instant startTime =
```

Comment:
if it's not a sumologic event (for example the Egress Event that I'm adding in RW-8410) then probably there's no need to do anything here. so if the optional is empty we can return directly...I think.

Reply:
ah, good point! Thanks!